packages/design-system/src/SegmentedControl/SegmentedControl.styles.tsx

```diff
@@ -7,10 +7,10 @@ export const StyledSegmentedControl = styled.div<{
   align-items: center;
   justify-content: space-between;
   gap: var(--ads-v2-spaces-2);
-  background-color: var(--ads-v2-color-bg-muted);
+  background-color: var(--ads-v2-color-bg-subtle);
   padding: var(--ads-v2-spaces-1);
-  border-radius: var(--ads-v2-border-radius);
-  height: 32px;
+  /* outer radius = inner radius + padding */
+  border-radius: calc(var(--ads-v2-border-radius) + var(--ads-v2-spaces-1));
```

Comment:
Let's put this in the theme variable file so that we can just reuse the calculation every time?